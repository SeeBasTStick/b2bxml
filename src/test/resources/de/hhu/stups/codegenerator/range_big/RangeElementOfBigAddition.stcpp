int main() {
    RangeElementOfBig range;
    range.simulate();
    cout << range.getCounter();
    return 0;
}