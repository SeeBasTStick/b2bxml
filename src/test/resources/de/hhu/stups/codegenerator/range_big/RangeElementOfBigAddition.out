public static void main(String[] args) {
    RangeElementOfBig range = new RangeElementOfBig();
    range.simulate();
    System.out.println(range.counter);
}